runbot_buildout/models/runbot_build.py

```diff
@@ -0,0 +1,335 @@
+# Copyright 2017-2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import glob
+import os
+import shutil
+import subprocess
+import tempfile
+from odoo import api, fields, models
+from odoo.tools import file_open
+from odoo.addons.runbot.common import lock, grep
+
+
+class RunbotBuild(models.Model):
+    _inherit = 'runbot.build'
+
+    @api.multi
+    def _server(self, *l, **kw):
+        self.ensure_one()
+        if self.repo_id.uses_buildout:
+            odoo_part = 'parts/%s' % self.repo_id.buildout_section
+            if os.path.exists(self._path(odoo_part, 'odoo')):
+                return self._path(odoo_part, 'odoo', *l, **kw)
+            return self._path(odoo_part, 'openerp', *l, **kw)
+        else:
```

Comment:
`else-return` is not needed if you `if` section uses a `return`
You can use just `return ...` without `else`

Reply:
What about use return early?

```python
if not  self.repo_id.uses_buildout:
    return ...
# My code for uses_buildout without nested `if` and without `else` reduce cyclomatic complex
```